zerver/models.py

```diff
@@ -269,6 +269,12 @@ class Meta:
         unique_together = ("realm", "name")
 
 
+def generate_realm_uuid_owner_secret() -> str:
+    token = generate_api_key()
+
+    return f"zuliprealm_{token}"
```

Comment:
I feel like it's worth a comment about why we're adding this prefix.

Reply:
True - added